.github/workflows/ci.yml

```diff
@@ -47,11 +47,16 @@ jobs:
         trino: [
           "latest",
         ]
+        sqlalchemy: [
+          "1.4"
```

Comment:
How about testing it always with the latest `1.3.x` and `1.4.x` version? It could help identify some issues earlier. Upstream client tools like superset use the latest patch.

Reply:
I don't see why matrix has `1.4`? Shouldn't it have `1.3` instead since setup.py currently installs latest `1.x`.

Reply:
> I don't see why matrix has 1.4? Shouldn't it have 1.3 instead since setup.py currently installs latest 1.x.

I added one extra include with latest 1.3. similar like we always test with latest trino except for some additional includes.

Reply:
> How about testing it always with the latest `1.3.x` and `1.4.x` version?

Fixed in latest commit